### PR TITLE
Discuss mid-less scenarios

### PR DESCRIPTION
**Pull request by juberti**

### CODE REVIEW

**Comment by juberti** [comment]
Fixes #325

**Comment by fluffy** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -631,10 +631,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         target="RFC5888"></xref>, Section 4, to identify
                         the m= line. JSEP implementations creating an
                         ICE Candidate object MUST populate both of these
-                        fields. Implementations receiving an ICE
+                        fields, using the MID of the associated RtpTransceiver
+                        object (which may be locally generated when interacting
+                        with a remote non-JSEP endpoint
+                        that does not support the MID attribute).
+                        Implementations receiving an ICE
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: This seems almost right. Maybe ("which may be locally generated by the answerer") and then link to the section below where you explain that.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -631,10 +631,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
                         Candidate object MUST use the MID if present, or
-                        the m= line index, if not (as it could have come
-                        from a non-JSEP endpoint).</t>
+                        the m= line index, if not (as in the aforementioned
+                        case).
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: I think I would spell this out "(e.g., when interacting with a non-JSEP remote endpoint)"

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: done

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: done

**Comment by juberti** [comment]
PTAL

**Pull request merged**